app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post
+
+  def self.update_like_counter(post)
+    post.update(likesCounter: post.likes.size)
+  end
```

Comment:
- [ ] Let us kindly implement same here :man_technologist:

Reply:
Thank you for your suggestion. I implemented this.